src/components/Admin/Page/Builder/Selector.vue

```diff
@@ -8,6 +8,7 @@
           <b-list-group-item
             v-for="(type) in types"
             :key="type.label"
+            :data-test-id="`select-${labelType(type.label)}-block`"
```

Comment:
No need for changes in this component, page block can be determined based on the button value